### ISSUE TITLE
Apple M1 support

### ISSUE THREAD

**Issue by drewwells** [czarny/brisync#40]
Looks like my dell u3818dw is not recognized on apple m1. It works great on Intel macbook. 

<img width="259" alt="image" src="https://user-images.githubusercontent.com/239123/116700326-439fe500-a98c-11eb-8613-b51f238ad4d6.png">

**Comment by czarny** [comment]
Would you be willing to share your machine and remote debug in order to examine the problem?

**Comment by drewwells** [comment]
Yeah we could setup something. Send me an email and we can arrange a time. My email is on my profile page.

**Comment by ghost** [comment]
Hey, same problem as drewwells but dell TN DELL SE2417HGX

**Comment by czarny** [comment]
It looks like it is hardware problem. No much to do :(

**Comment by borekb** [comment]
M1 doesn't support DDC, this is how Lunar handles it: https://lunar.fyi/#m1 (TL;DR: either gamma tables which is not great or a _very_ involved but clever Raspberry Pi workaround 😄).

**Comment by drewwells** [comment]
How does m1 control tb3 monitors?

**Comment by hau21um** [comment]
Lunar now works properly so seems somehow doable.

**Comment by akshaysalunke13** [comment]
Same issue on m1 pro Mac. Lunar works fine. 

@czarny Let me know if I could be of some help here.

**Comment by czarny** [comment]
Hi, I have working prototype which handles Apple silicone but there is still a bunch of work to do. Expect an update 'soon'.

**Comment by czarny** [comment]
It took more than I expected, but it is done :) Feel free to download Brisync 2.0.0 with apple silicon macs suport.
